api/tests/integration/infrastructure/repositories/student-repository_test.js

```diff
@@ -0,0 +1,37 @@
+const { expect, databaseBuilder } = require('../../../test-helper');
+const studentRepository = require('../../../../lib/infrastructure/repositories/student-repository');
+
+describe('Integration | Infrastructure | Repository | student-repository', () => {
+
+  describe('#findReconciledStudentsByNationalStudentId', () => {
+
+    it('should return instances of Student', async () => {
+      // given
+      const firstNationalStudentId = '1234';
```

Comment:
Peut être overkill, mais pourquoi pas utiliser [le format officiel](https://fr.wikipedia.org/wiki/Num%C3%A9ro_BEA)
```suggestion
      const firstNationalStudentId = '12345789AB';
```